data/network/src/iosTest/kotlin/social/androiddev/common/ResourceLoader.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of Dodo.
+ *
+ * Dodo is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * Dodo is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Dodo. If not, see <https://www.gnu.org/licenses/>.
+ */
+package social.androiddev.common
+
+import kotlinx.cinterop.addressOf
+import kotlinx.cinterop.usePinned
+import platform.Foundation.NSBundle
+import platform.Foundation.NSData
+import platform.Foundation.NSUTF8StringEncoding
+import platform.Foundation.dataWithContentsOfFile
+import platform.posix.memcpy
+
+actual fun readBinaryResource(resourcePath: String): ByteArray {
+    val path = resourcePath.substringBeforeLast(".")
+    val fileType = resourcePath.substringAfterLast(".")
+    println("resourcePath=$resourcePath, path=$path, fileType=$fileType")
```

Comment:
Although it's just test, probably can remove debug print statements in this file

Reply:
I left it for debugging since it is not due to a flaky CI. It is currently not working for iOS / native.
I read there is another gradle task needed to copy the assets into the build directory.

Reply:
You mean the copy task from the article or a different one?

Reply:
Yes, I think I haven't added it already.